components/Header.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import HeaderImage from './assets/header.jpg'
```

Comment:
Capitalised names are reserved for components, not images

Reply:
& what is recommended to use for images? :)

Reply:
`headerImage` is enough :) lowercase names are standard